pkg/mountinfo/mountinfo.go

```diff
@@ -19,14 +19,17 @@ import (
 	"fmt"
 	"io"
 	"os"
+	"path/filepath"
 	"strconv"
 	"strings"
+
+	"golang.org/x/sys/unix"
 )
 
 const (
-	// FilesystemType names for filesystem which are used in /proc/pid/mountinfo
-	FilesystemTypeBPFFS   = "bpf"
-	FilesystemTypeCgroup2 = "cgroup2"
+	// FilesystemType supeblock magic numbers for filesystems
+	FilesystemTypeBPFFS   = unix.BPF_FS_MAGIC
+	FilesystemTypeCgroup2 = unix.CGROUP2_SUPER_MAGIC
```

Comment:
These are only used in tests now AFAICS. Could we move them to `pkg/mountinfo/mountinfo_test.go`?

Reply:
`FilesystemTypeCgroup2` even looks entirely unused.

Reply:
Not really. It is not seen from the diff (because it hasn't changed) but it is used by whoever is calling `mountinfo.IsMountFS()`:


```
pkg/bpf/bpffs_linux.go:176:	mounted, bpffsInstance, err := mountinfo.IsMountFS(mountinfo.FilesystemTypeBPFFS, bpfRoot)
pkg/bpf/bpffs_linux.go:216:	mounted, bpffsInstance, err := mountinfo.IsMountFS(mountinfo.FilesystemTypeBPFFS, mapRoot)
pkg/bpf/bpffs_linux.go:249:		cMounted, cBpffsInstance, err := mountinfo.IsMountFS(mountinfo.FilesystemTypeBPFFS, mapRoot)
pkg/cgroups/cgroups.go:78:	mounted, cgroupInstance, err := mountinfo.IsMountFS(mountinfo.FilesystemTypeCgroup2, cgroupRoot)
```

Reply:
Alternatively, I can rename these two (`s/FilesystemType/FSType/` or so) and the diff will show all the users. I don't think it makes sense though other than for illustrative purposes.

Reply:
You're right, no need to rename. Thanks.